doc/tutorials/objdetect/traincascade.markdown

```diff
@@ -287,13 +287,18 @@ Command line arguments of opencv_traincascade application grouped by purposes:
 
     -   -minHitRate \<min_hit_rate\>
 
-        Minimal desired hit rate for each stage of the classifier. Overall hit rate may be estimated
-        as (min_hit_rate\^number_of_stages).
+        Minimal desired hit rate for each stage of the classifier. The overal hit rate can be calculated as follows.
+
+        overall_hit_rate = 1.0 - overall_fail_rate
+        overall_fail_rate  = max_fail_rate ^ number_of_stages
+        max_fail_rate = 1.0 - min_hit_rate
+
+        and thus overall_hit_rate = ( 1.0 - ( 1.0 - min_hit_rate ) ^ number_of_stages )
```

Comment:
it's not quite clear for me. If each stage has at least 0.99 hit-rate, when the cascade of 20 stages will have 0.99^20 hit-rate, right? that is, formula should be

        overall_hit_rate = min_hit_rate ^ number_of_stages